base_geoengine/geo_point.py

```diff
@@ -18,16 +18,18 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from __future__ import absolute_import
-from osv import fields
+
+from openerp.osv import fields
 from . import geo_field
 
+
 class GeoPoint(geo_field.Geom):
     """New type of column in the  ORM for POSTGIS geometry Point type"""
     _type = 'geo_point'
-    def __init__(self, string, dim=2, srid=900913 , gist_index=True, **args):
-        res = super(GeoPoint, self).__init__(string, "POINT", dim=dim,
-                                             srid=srid, gist_index=gist_index, **args)
-        return res
+    _geo_type = 'POINT'
+
+    def __init__(self, string, dim=2, srid=900913, gist_index=True, **args):
+        super(GeoPoint, self).__init__(
```

Comment:
Inheriting the docstring from super can help document what the parameters are supposed to be.